.github/workflows/reusable-api-e2e.yml

```diff
@@ -61,15 +78,27 @@ jobs:
       - name: Build API
         run: CI='' pnpm build:api
 
+
       - name: Run E2E tests
         run: |
           cd apps/api && pnpm test:e2e
 
+      - uses: mansagroup/nrwl-nx-action@v3
```

Comment:
we will run this one only if affected and ee matrix.

small note the default is true meaning the on deploy it is affected by default.